src/charts/grouped-bar.js

```diff
@@ -214,6 +217,38 @@ define(function (require) {
                 .attr('transform', `translate(${yTickTextOffset['x']}, ${yTickTextOffset['y']})`);
         }
 
+        /**
+         * Gets or Sets the padding between bars.
+         * The default value is 0.1
+         * @param  {Number} _x Padding value to get/set
+         * @return {Number | module} Current padding or Chart module to chain calls
+         * @public
+         */
+        exports.betweenBarsPadding = function (_x) {
+            if (!arguments.length) {
+                return betweenBarsPadding;
+            }
+            betweenBarsPadding = _x;
+
+            return this;
+        };
+
+        /**
+         * Gets or Sets the padding between groups of bars.
+         * The default value is 0.1
+         * @param  {Number} _x Padding value to get/set
+         * @return {Number | module} Current group padding or Chart module to chain calls
+         * @public
+         */
+        exports.betweenGroupsPadding = function (_x) {
+            if (!arguments.length) {
+                return betweenGroupsPadding;
+            }
+            betweenGroupsPadding = _x;
+
+            return this;
+        };
+
```

Comment:
This API related code needs to go down in the file, by alphabetic order if possible :)

Reply:
Yup! my bad @Golodhros ; Make me know if anything is still wrong